L1Trigger/L1CaloTrigger/plugins/Phase2L1CaloEGammaEmulator.h

```diff
@@ -0,0 +1,207 @@
+//------------------------------------
+// Helper functions for Phase2L1CaloEGammaEmulator.h
+//------------------------------------
+
+#include <ap_int.h>
+#include <fstream>
+#include <iomanip>
+#include <iostream>
+#include <stdio.h>
+
+#ifndef EGAMMA_CRYSTALS_PRODUCER_H
+#define EGAMMA_CRYSTALS_PRODUCER_H
+
+static constexpr int dimPhi = 4;
+static constexpr int dimEta = 17;
+static constexpr int dimCard = 36;
+static constexpr int nClustersPerLink = 2;
```

Comment:
These specific constants were deleted because they are unused, but other const ints are now associated with namespace in commit [1f84a28](https://github.com/cms-l1t-offline/cmssw/pull/1069/commits/1f84a2865fab8bebb201239efab0ccf9477d46ad)